TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/HostApplicationBuilderExtensions.cs

```diff
@@ -101,6 +102,15 @@ public static IHostApplicationBuilder AddBackgroundJobs(this IHostApplicationBui
                     job => job.Execute(),
                     Cron.Never);
 
+                var giasOptions = sp.GetService<IOptions<GiasOptions>>();
+                if (giasOptions != null)
```

Comment:
Do we need this null check?

Reply:
that's what I'm using to not add the job